tools/operator/testdata/kafka-leveldb-notls-network-spec.yml

```diff
@@ -5,7 +5,7 @@
 #! fabricVersion:
 #! Released images are pulled from docker hub hyperledger/, e.g. 1.4.1 or 2.0.0
 #! Development stream images are pulled from
-#! nexus3.hyperledger.org:10001/hyperledger/, e.g. 1.4.1-stable or 2.0.0-stable
+#! hyperledger-fabric.jfrog.io/, e.g. 1.4.1-stable or 2.0.0-stable
 fabricVersion: 2.0.0-stable
```

Comment:
It seems we are publishing images with "latest" tag. We also have similar files in release-1.4 branch. Publishing with "latest" tag in 2.0 branch and 1.4.5-stable for 1.4 branch is fine.

Reply:
Correct, I need to do this work in 1.4 as well. We will publish stable and latest in master, and only stable in 1.4

Reply:
in master, is latest for the dev branch (post-merge), and, the 2.0.0-stable for the post-interop integratioin branch?

Reply:
@scottz64 This doesn't bring any of the interop methodology into play yet, this just replicates what we have today